.github/workflows/test_simple.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# fail on non-zero return code from a subprocess
+set -e
+
+# print commands
+set -x
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 GDAL_AUTOLOAD_DIR"
+    exit 1
+fi
+
+# non-existent variables as an errors
+set -u
+
+export GDAL_AUTOLOAD_DIR=$1
+export GDAL_DRIVER_PATH=${GDAL_AUTOLOAD_DIR}
+
+# add small GRASS GIS dataset for tests
+(mkdir -p $HOME/grassdata && \
+ cd $HOME/grassdata/ && \
+ wget -c --quiet https://grass.osgeo.org/sampledata/north_carolina/nc_spm_08_micro.zip && \
+ unzip nc_spm_08_micro.zip && \
+ rm -f nc_spm_08_micro.zip )
+
+# workaround for GRASS GIS < 7.8.8
+export LD_LIBRARY_PATH=$(pkg-config --libs-only-L grass | sed 's+-L++g')
```

Comment:
I see this (`$(pkg-config --libs-only-L grass | sed 's+-L++g')`) is resolved as `LD_LIBRARY_PATH=/usr/lib/grass7.8/lib`

shouldn't it be `LD_LIBRARY_PATH=/usr/lib/grass78/lib` (without dot)?

Reply:
If it is certain it is G78 then
`LD_LIBRARY_PATH=$(grass78 --config path)` could be an alternative.

Reply:
Oh right, I overlooked the dot!

Reply:
IMHO, a generic way to set `LD_LIBRARY_PATH` that works with both GRASS 7 and GRASS 8 should be used if possible.

Reply:
> IMHO, a generic way to set `LD_LIBRARY_PATH` that works with both GRASS 7 and GRASS 8 should be used if possible.

Absolutely. As `grass.pc` seem to be broken an alternative is: `grass --config path` which seems to work. But pre-G8 it is `grass78 --config path` etc.

Reply:
Success! And yes, changed now to "versionless" with `grass` as well.

https://github.com/neteler/gdal-grass/actions/runs/2260092078

Thanks for your assistance!

Reply:
> Success! And yes, changed now to "versionless" with `grass` as well.
> 
> https://github.com/neteler/gdal-grass/actions/runs/2260092078
> 
> Thanks for your assistance!

Great! Is `grass78` symlinked to `grass` on Linux?

Reply:
> Great! Is `grass78` symlinked to `grass` on Linux?

At least on Debian, Ubuntu and Fedora yes.